analysis_options.yaml

```diff
@@ -10,29 +11,69 @@ linter:
   rules:
     - annotate_overrides
     - avoid_empty_else
+    - avoid_function_literals_in_foreach_calls
```

Comment:
This is a lot of lints.  Is this a group you've used elsewhere?  If it's shared, maybe we should consider putting the set someplace shareable the way package `pedantic` does?

Reply:
Yeah...this is a group of things we've been pushing around through a lot of packages I work with – build and friends.

Reply:
Have you considered capturing them in one place?  How do you maintain them?  Is there any discussion we could link to somewhere?

Reply:
Not really...although it wouldn't be a bad idea.